packages/@tinacms/graphql/src/resolver/index.ts

```diff
@@ -64,12 +67,15 @@ export class Resolver {
     // if (res.type === "union") {
     //   extraFields["templates"] = res.templates;
     // }
-    let documents = {}
-    if (hasDocuments) {
-      documents = await this.getDocumentsForCollection(collectionName)
-    }
+    // let documents = {}
```

Comment:
I'm confused by this - I thought the commented out code here was the fix for the issue you mentioned